github-dark.css

```diff
@@ -2898,10 +2898,10 @@
   .select-menu-item.navigation-focus > .octicon,
   .select-menu-item[aria-selected="true"] > .octicon, .text-white,
   .octicon.text-white,
-  .select-menu-item.selected.navigation-focus .description-warning,
   .suggester li.navigation-focus, .suggester li:hover, .suggester li:hover *,
   .suggester li[aria-selected="true"], .suggester li[aria-selected="true"] *,
-  details-menu .select-menu-item[aria-checked=true]:hover .description {
+  details-menu .select-menu-item[aria-checked=true]:hover .description,
+  .select-menu-item.navigation-focus .select-menu-item-text .description strong {
```

Comment:
Instead of adding `strong` to the selector it's probably better to move this rule to after the red one.

Reply:
You're right this requires more thought, the milestone being selected/non-selected matters as both need to be accounted for to resolve full issue.
Also position.

Reply:
Ive had a look, I cant get this to work unless the selector is changed, as for position this is well far past the red one as is.

Ive updated the selector a bit and still works, failing this solution, ive no idea.